Index: src/ocgis/calc/base.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/ocgis/calc/base.py	(revision fe57ecb93efe574ca122e61cbf1569e02bed6cdd)
+++ src/ocgis/calc/base.py	(revision 857a0c99e5bd06758986f5ad76d32499f7bb644c)
@@ -116,7 +116,7 @@
         :rtype: :class:`numpy.ma.core.MaskedArray`
         """
 
-        ret = np.ma.average(values, weights=weights)
+        ret = np.ma.average(np.squeeze(values), weights=weights)
         return ret
 
     def aggregate_temporal(self, values, **kwargs):
@@ -132,7 +132,7 @@
     @abc.abstractmethod
     def calculate(self, values, **kwargs):
         """
-        The calculation method to overload. Values are explicitly passed to  avoid dereferencing. Reducing along the
+        The calculation method to overload. Values are explicitly passed to avoid dereferencing. Reducing along the
         time axis is required (i.e. axis=0).
 
         :param values: A three-dimensional array with dimensions (time, row, column).
